internal/cli/atlas/clusters/delete.go

```diff
@@ -81,10 +109,16 @@ Deleting a cluster also deletes any backup snapshots for that cluster.
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return opts.Run()
 		},
+		PostRunE: func(cmd *cobra.Command, args []string) error {
+			return opts.PostRun()
+		},
 	}
 
 	cmd.Flags().BoolVar(&opts.Confirm, flag.Force, false, usage.Force)
 
+	cmd.Flags().BoolVarP(&opts.EnbaleWatch, flag.EnableWatch, flag.EnableWatchShort, false, usage.EnableWatch)
+	cmd.Flags().UintVar(&opts.Timeout, flag.WatchTimeout, 0, usage.WatchTimeout)
```

Comment:
Does this override the default timeout in the watcher? `const defaultWait = 4 * time.Second`

Reply:
Not yet - this will be implemented as part of this ticket: CLOUDP-181597. I was thinking that the watch command would still get the resource every 4 seconds for a linear backoff, but it will end once it's been trying for `opts.Timeout` amount of time.